README.md

```diff
@@ -57,3 +57,25 @@ Please star them if you are experiencing them in your project.
 * DexFileDependenciesTask is not cacheable: https://issuetracker.google.com/160138798
 * MergeResources is not relocatable: https://issuetracker.google.com/issues/141301405
 * Room annotation processor causes cache misses, doesn't declare outputs, overlapping outputs, etc: https://issuetracker.google.com/issues/132245929
+
+## Implementation Notes
+
+### RoomSchemaLocationWorkaround
+
+Most of the workarounds in this plugin should apply to your project without any changes.  However, one workaround
+requires some extra configuration.  The Room schema location workaround allows you to specify an output directory for
+Room schema exports without breaking caching for your Java and/or Kapt tasks where the annotation processor has been configured.
+There are various issues with how this annotation processor works (see https://issuetracker.google.com/issues/132245929
```

Comment:
Having this pointed out in the readme is great as it will make people star the bug against google!